site/content/docs/0.0/utilities/borders.md

```diff
@@ -8,4 +8,293 @@ aliases:
+
+### How it works
+
+Consider our default `.border-success` utility.
+
+```css
+.border-success {
+  --bs-border-opacity: 1;
+  border-color: rgba(var(--bs-success-rgb), var(--bs-border-opacity)) !important;
+}
+```
+
+We use an RGB version of our `--bs-success` (with the value of `25, 135, 84`) CSS variable and attached a second CSS variable, `--bs-border-opacity`, for the alpha transparency (with a default value `1` thanks to a local CSS variable). That means anytime you use `.border-success` now, your computed `color` value is `rgba(25, 135, 84, 1)`. The local CSS variable inside each `.border-*` class avoids inheritance issues so nested instances of the utilities don't automatically have a modified alpha transparency.
+
+### Example
+
+To change that opacity, override `--bs-border-opacity` via custom styles or inline styles.
+
+{{< example >}}
+<div class="border border-success p-2">This is default success border</div>
+{{< /example >}}
+
+<details>
+<summary>See Bootstrap examples that are incompatible with Orange Design System.</summary>
+<br>
+{{< design-callout-alert >}}
+These border color variants **with opacity** should not be used because they do not respect the Orange Design System specifications.
+{{< /design-callout-alert >}}
+
+{{< example >}}
+<div class="border border-success p-2 mb-2">This is default success border</div>
+<div class="border border-success p-2" style="--bs-border-opacity: .5;">This is 50% opacity success border</div>
+{{< /example >}}
+
+Or, choose from any of the `.border-opacity` utilities:
+
+{{< example >}}
+<div class="border border-success p-2 mb-2">This is default success border</div>
+<div class="border border-success p-2 mb-2 border-opacity-75">This is 75% opacity success border</div>
+<div class="border border-success p-2 mb-2 border-opacity-50">This is 50% opacity success border</div>
+<div class="border border-success p-2 mb-2 border-opacity-25">This is 25% opacity success border</div>
+<div class="border border-success p-2 border-opacity-10">This is 10% opacity success border</div>
+{{< /example >}}
+</details>-->
+
+## Width
+
+{{< example class="bd-example-border-utils" >}}
+<span class="border border-none"></span>
+<span class="border border-default"></span>
+<span class="border border-thick"></span>
+<span class="border border-thicker"></span>
+<span class="border border-thickest"></span>
+{{< /example >}}
+
+{{< bootstrap-compatibility >}}
+
+{{< example class="bd-example-border-utils" >}}
+<span class="border border-0"></span>
+<span class="border border-1"></span>
+<span class="border border-2"></span>
+<span class="border border-3"></span>
+<span class="border border-4"></span>
+<span class="border border-5"></span>
+{{< /example >}}
+
+{{< /bootstrap-compatibility >}}
+
+## Radius
+
+Add classes to an element to easily round its corners.
+
+<!-- TODO : This example with default rounded value doesn't have any sens since the default value is 0. Do we keeo it ? -->
```

Comment:
since `default`is set to 0, the `rounded` class doesn't add any border-radius. This also applies to `rounded-top` and such. In another brand this could be usefull but not in OUDS. Do we drop this or explain that border-radius are not allowed by default. This default management may need to be reviewed since it also generate a CSS variable.